### DISCUSSION TITLE
Backward button misbehaving on Firefox Android

### DISCUSSION THREAD

**Discussion by jeesusonherraturku** [community/community#40554] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

Backward button misbehaving on Firefox Android when browsing trending repositories on Explore. 

If you click on a repository on trending list, and then try to click Back it will first jump to bottom of the page and them second clicking will take you to front page of explore and it reads

"Here's what we found based on your interests... "

It should take you back to trending list where you came from. Now I need to press trending again.

It seems to work on chrome Android. 

Thank you.

**Comment by ossanna16** [comment]
Hi there @jeesusonherraturku and welcome to our community! Thank you for reporting this!

**Reply by jeesusonherraturku** [reply]
It seems to work now properly. Thank you.

**Reply by ossanna16** [reply]
Glad to hear that!
